packages/backend/src/api/routers/StatusRouter.ts

```diff
@@ -1,19 +1,19 @@
 import Router from '@koa/router'
 
-import { StatusController } from '../controllers/StatusController'
+import { StatusController } from '../controllers/status/StatusController'
 
 export function createStatusRouter(statusController: StatusController) {
   const router = new Router()
 
-  router.get('/api/status/prices', async (ctx) => {
+  router.get('/status/prices', async (ctx) => {
```

Comment:
Not urgent, but I guess we'd like to have basicauth for all these endpoints on prod? cc @sz-piotr